### PR TITLE
changed code to print combined pdf statements

### PR DESCRIPTION
**Pull request by venambati**
Hello Brady,

Here is the fix to billing statements.

when we goto Fees->postings and select patients with multiple encounters with balances 5.0.1 codes is 
printing 2 statements one for each encounter. This code fixes that.

Test cases Tested with:

1. one patient with one encounter/fees
2. one patient with multiple encounters/fees
3. multiple patients with multiple encounters/fees

please review and let me know.

Thank you
Venkat

### CODE REVIEW

**Comment by bradymiller** [comment]
Hi @venambati , It looks like you are removing some unrelated code.

**Comment by bradymiller** [comment]
oops, didn't mean to close this (I just reopened it) :)

**Comment by venambati** [comment]
@bradymiller I will go over the changes very carefully again, the code I removed is being replaced by the new code. I But I am going to review my changes again.

Thank you
Venkat

**Comment by bradymiller** [comment]
@venambati ,
Check out this link which ignores white space:
https://github.com/openemr/openemr/pull/790/files?w=1

File: interface/billing/sl_eob_search.php [hunk 1]
```diff
@@ -406,6 +406,7 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
     //
 
    if ($stmt['cid'] != $row['pid']) {
+        if (!empty($stmt)) ++$stmt_count;
```
Comment (robertdown) [inline interface/billing/sl_eob_search.php]: Use curly braces and place the iterator on a separate line per PSR standards

File: interface/billing/sl_eob_search.php [hunk 2]
```diff
@@ -424,11 +425,23 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
...
         $stmt['patient'] = $row['fname'] . ' ' . $row['lname'];
+
+        $stmt['encounter'] = $row['encounter'];
+  		#If you use the field in demographics layout called
+  		#guardiansname this will allow you to send statements to the parent
+  		#of a child or a guardian etc
+      if(strlen($row['guardiansname']) == 0) {
         $stmt['to'] = array($row['fname'] . ' ' . $row['lname']);
+      } else
+      {
```
Comment (robertdown) [inline interface/billing/sl_eob_search.php]: Move to above line with `} else` per PSR standards

**Comment by bradymiller** [comment]
@venambati ,

I just wanted to confirm that these are your wished code changes(added flag to not show whitespace changes):
https://github.com/openemr/openemr/pull/790/files?w=1

thanks,
-brady

**Comment by venambati** [comment]
@bradymiller yes, these are my changes for the feedback by @robertdown to comply with PSR standards on curly braces.

Thank you

File: interface/billing/sl_eob_search.php [hunk 3]
```diff
@@ -404,40 +404,59 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
...
+                upload_file_to_client_email($stmt['pid'], $STMT_TEMP_FILE);
+                fclose($fhprint);
+                sleep(1);
+                $fhprint = fopen($STMT_TEMP_FILE, 'w');
+                $stmt = array();
+        }
+
+      //if ($stmt['cid'] != $row['pid']) {
+        //if (!empty($stmt)) ++$stmt_count;
+        //fwrite($fhprint, create_pdf_statement($stmt));
```
Comment (bradymiller) [inline interface/billing/sl_eob_search.php]: remove code rather than comment it out

File: interface/billing/sl_eob_search.php [hunk 4]
```diff
@@ -404,40 +404,59 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
...
+        $stmt['cid'] = $row['pid'];
+        $stmt['pid'] = $row['pid'];
+                $stmt['dun_count'] = $row['stmt_count'];
+                $stmt['bill_note'] = $row['pat_billing_note'];
+                $stmt['enc_bill_note'] = $row['enc_billing_note'];
+                $stmt['bill_note'] = $row['billing_note'];
```
Comment (bradymiller) [inline interface/billing/sl_eob_search.php]: did you mean to add the line above here (note it overwrite the $stmt['bill_note'] 3 lines above this.

**Pull request closed without merging**